packages/terra-form/src/components/ChoiceField.jsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable jsx-a11y/label-has-for */
+
```

Comment:
Yes, this allows them to be loaded independently without needing to be within the `<Base>` component.